src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/containerregistry/container_registry.py

```diff
@@ -0,0 +1,84 @@
+#---------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#---------------------------------------------------------------------------------------------
+#pylint: skip-file
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Code generated by Microsoft (R) AutoRest Code Generator 0.16.0.0
+# Changes may cause incorrect behavior and will be lost if the code is
+# regenerated.
+# --------------------------------------------------------------------------
+
+from msrest.service_client import ServiceClient
+from msrest import Configuration, Serializer, Deserializer
+from .version import VERSION
+from .operations.registries import Registries
+from . import models
+
+
+class ContainerRegistryConfiguration(Configuration):
```

Comment:
Azure container registry is a different service from compute or ACS. ACS is one consumer of the registry and container registry can be used without Azure VMs or websites. We have deployed from this registry into other cloud providers and local network machines, and we regularly use it as part of our development process via the CI containers.